app/controllers/guest_rooms_controller.rb

```diff
@@ -0,0 +1,6 @@
+class GuestRoomsController < ApplicationController
+  def create
+    guest = GuestRoom.create!(guest_id: params[:id], room_id: params[:room_id])
+    redirect_to "/guests/#{params[:id]}"
+  end
```

Comment:
One thing to be careful of with the ! on line 3 is that your code will throw a gross user-side error (on the browser) if for whatever reason that GuestRoom couldn't be created (maybe they went in an invalid room id or something). Those bangs (!) can be super helpful in tests, OR they are helpful when you have exception handlers. If you're interested, take a look at exception handlers to see how you can utilize forced errors in your code.